vivarium/core/process.py

```diff
@@ -143,29 +151,157 @@ def __init__(self, parameters: Optional[dict] = None) -> None:
         elif not hasattr(self, 'name'):
             self.name = self.__class__.__name__
 
-        self.parameters = copy.deepcopy(self.defaults)
-        self.parameters = deep_merge(self.parameters, parameters)
-        self.schema_override = self.parameters.pop('_schema', {})
-        self.parallel = self.parameters.pop('_parallel', False)
-        self.condition_path = None
+        self._parameters = copy.deepcopy(self.defaults)
+        self._parameters = deep_merge(self._parameters, parameters)
```

Comment:
we use `self.parameters` all over, not sure this `_` to make it private will be backwards-compatible.